packages/picasso-charts/src/BarChart/story/index.jsx

```diff
@@ -105,3 +111,8 @@ page
     },
     delay: 500
   })
+  .addExample('BarChart/story/BarLabelHidden.example.tsx', {
+    title: 'Hidden bar label',
+    description: 'You can hide label of each bar via `barLabelHidden` prop.',
+    delay: 500
+  })
```

Comment:
I would suggest skipping visuals for jest and using happoSnapshot in cypress side

```suggestion
  }) // picasso-skip-visuals
```